src/dune_rules/melange/melange_rules.ml

```diff
@@ -235,6 +233,7 @@ let setup_entries_js ~sctx ~dir ~dir_contents ~scope ~compile_info ~target_dir
     Dir_contents.ocaml dir_contents
     >>| Ml_sources.modules_and_obj_dir ~for_:(Melange { target = mel.target })
   in
+  let obj_dir = Obj_dir.of_local obj_dir in
```

Comment:
This is more a code style question. I think I had added `obj_dir` inside the `parallel_iter` callback below to limit its scope, wouldn't this change go against the code style guidelines? I can't see any new usages of it.

Reply:
Just because it's clearer if it's only computed once rather than once per iteration of the loop. I'll move it closer to the usage though.